src/site/xdoc/google_style.xml

```diff
@@ -1032,6 +1032,15 @@
                   config</a>)
                   <br />
                   <br />
+                  A single whitespace before the empty block(<code>{}</code>) of a class,
+                  method, interface, static, loops(<code>while</code>,
+                  <code>do</code>, <code>for</code>), and record is not covered.
+                  It will be addressed at:
+                  <a href="https://github.com/checkstyle/checkstyle/issues/17715">
```

Comment:
We have blue colors for cases when there bunch of edge cases that we will not cover at all.

Let's not make this table a reflection of issue tracking system.
We will never catch up.

Issue is to wording for cases we can not support, so it is partial support.